examples/lets.kl

```diff
@@ -0,0 +1,14 @@
+#lang kernel
+
+(example (the Syntax (let (x 'foo) x)))
+
+(example (flet (f (x) (if x 'a 'b)) (f #true)))
+
+(example (the Syntax (flet (f (x) 'a) (list-syntax ((f #true) (f 4)) 'here))))
+
+(example (the Syntax (let (f (lambda (x) 'a)) (list-syntax ((f #true) (f 4)) 'here))))
+
+(define id (lambda (x) x))
+
+(example (the Syntax (id 'foo)))
+(example (the Signal (id 45)))
```

Comment:
It would be nice to have an example `the T` for every primitive type `T` in a separate file (perhaps `these.kl`)

Reply:
Ah, looks like that's in `tiny-types.kl`. These can probably be moved there? What's the difference between the content of that file and this one?

Reply:
Only a chronological difference WRT when I wrote it. It's not a thought-through placement.